authors/apps/profiles/models.py

```diff
@@ -0,0 +1,38 @@
+from django.db import models
+from authors.apps.authentication.models import User
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+class Profile(models.Model):
+    
+    # one to one relationship
+    user = models.OneToOneField(
+        User, on_delete=models.CASCADE
+    )
+    firstname = models.CharField(max_length=250, default='')
+    lastname = models.CharField(max_length=250, default='')
+    # display some info about a user
+    bio = models.TextField(blank=True)
+    username = models.CharField(max_length=50, unique=True)
```

Comment:
Having a username in the profile table creates data redundancy as we already have the username in the User table.

Reply:
Done Working On it .